### PR TITLE
feat: [PagoPa-1372] int test multiple debt pos

### PR DESCRIPTION
**Pull request by alessio-acitelli**
#### List of Changes

- added integration tests for the bulk upload function

#### Motivation and Context

- verify the correct functioning of the bulk API

#### How Has This Been Tested?

- local run
- [DEV env integration tests](https://github.com/pagopa/pagopa-debt-position/actions/runs/7460066056)

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: integration-test/src/features/debt-position-multiple-create.feature [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Feature: Managing a debt position
+
+  Background:
+    Given GPD running
+
+  Scenario: Multiple Debt position creation
+    Given a random iupd
+    When the debit position items is created
```
Comment (pasqualespica) [inline integration-test/src/features/debt-position-multiple-create.feature:8]: typo `debit`

File: integration-test/src/step_definitions/support/clients/gpd_client.js [hunk 2]
```diff
@@ -16,7 +19,20 @@ function createDebtPosition(orgId, body, segCodes){
...
+            "Content-Type": "application/json"
+        }
+    })
+}
+
+function createMassiveDebtPositions(orgId, body, segCodes){
+	const params = {}
+	if (segCodes) {params.segregationCodes = segCodes}
+    return post(GPD_HOST_V2 + GPD_API_MASSIVE(orgId), body, {
+        timeout: 20000,
```
Comment (pasqualespica) [inline integration-test/src/step_definitions/support/clients/gpd_client.js:35]: could be set `timeout` as env param ?

**Review by pasqualespica — CHANGES_REQUESTED** [review]
ok , some little changes

File: integration-test/src/features/debt-position-multiple-create.feature (hunk 1 above)
Comment (alessio-acitelli) [reply integration-test/src/features/debt-position-multiple-create.feature:8]: done

File: integration-test/src/step_definitions/support/clients/gpd_client.js (hunk 2 above)
Comment (alessio-acitelli) [reply integration-test/src/step_definitions/support/clients/gpd_client.js]: done

**Review by FedericoRuzzier — APPROVED** [review]
LGTM 👍

**Pull request merged**